xarray/core/ops.py

```diff
@@ -517,3 +527,9 @@ def inject_bottleneck_rolling_methods(cls):
         func.__name__ = 'median'
         func.__doc__ = _ROLLING_REDUCE_DOCSTRING_TEMPLATE.format(name=func.__name__)
         setattr(cls, 'median', func)
+
+        # bottleneck doesn't offer rolling_count, so we construct it ourselves
+        func = rolling_count
+        func.__name__ = 'count'
+        func.__doc__ = _ROLLING_REDUCE_DOCSTRING_TEMPLATE.format(name=func.__name__)
+        setattr(cls, 'count', func)
```

Comment:
Is this indented too far.  Don't you want this method if we don't have bottleneck as well? I think this is causing your current test failure.